README.md

```diff
@@ -722,50 +669,22 @@ This means that you can carry on splitting as much as you like.
 const var [[[getScore, setScore], setScore], setScore] = use(0)!
 ```
 
-## AI
```

Comment:
Don't change the name Automatic insertion works too, imagine you need to insert a value or a string and not code, keep AI AEMI ABI AQMI